tests/run-tests

```diff
@@ -189,7 +189,12 @@ def run_micropython(pyb, args, test_file):
 
     return output_mupy
 
-def run_tests(pyb, tests, args):
+
+def run_feature_check(pyb, args, test_file):
```

Comment:
I thought you might have also passed "base_path" as an arg, and added the "feature_check/" dir in this function, so you could call it like: `run_feature_check(pyb, args, base_path, 'native_check.py')`